### ISSUE TITLE
Mailansicht - 2 Probleme - Versatz und Anzeige?

### ISSUE THREAD

**Issue by Christian-H** [yakamara/redaxo_prozer#198]
A) Es gibt den Fall, dass  Mails gelöscht werden und wenn man dann unten sich per klick die 2 Seite ansieht, dann fehlen aufgrund des Versatzes Mails. Bekanntes Problem

b) Auch wenn man keine Mails löscht und dann sich Seite 2 und 3 ansieht, werden nicht allsMails angezeigt. Nachdem man 1 bis zweimal auf Inbox klickt, tauchen auf einmal die Mails auf. 
Bsp. Abends verlässt man die Arbeit mit 10 ungelesenen Mails. Morgens macht man den Prozer auf und es stehen 23 ungelesene E-Mails drin. Doch diese werden nicht alle (gleich?) angezeigt. 
D.h. oben der Zähler stimmt, aber die Inbox Anzeige nicht. 
Dies war in der Redaxo Version 5 nicht so.

P.S.: Wir haben ja beim versenden Probleme, dass es ca. 5-7 mal pro Tag bei unterschiedlichen Usern zum Fehler kommt. Kann es da auch einen Abruffehler geben?

**Comment by Christian-H** [comment]
Also heute und gestern hatte ich Mails um 14:20 , wo die Mail ca. 45 Min. oder mehr verspätet auftauchte und heute um 14:27 eine Mail, die ich um 17 Uhr das erste mal sah. Das müssen wir uns anschauen.

**Comment by dergel** [comment]
prüfen und absprechen

**Comment by tbaddade** [comment]
Ich hatte schon den Fall, dass manche E-Mails doppelt auftauchten. Mal auf Seite 1 und dann auch noch auf Seite 2.

**Comment by dergel** [comment]
a) klar.. das ist in einem anderen issue - später mal - als thema.. 
b) wenn man den rechner verlässt, kann es passieren dass durch ruhezustand und verzögertes Laden hier und da ein zeitlicher Versatz ist. Das liegt an der bisherigen Umgebung und den Javascriptprozessen, die ein ganz eigenes Problem sind (Frameworkfrage) .. Das Problem kann meines Erachtens bei REDAXO 5 nur genauso gewesen sein (Vielleicht so nicht aufgefallen)

@Christian-H Das "P.S.:" meint genau diese Probleme a,b ?

Das können wir sie nicht ohne größeren Aufwand und Umstellung ändern. Soweit ich das Problem verstehe

**Comment by Christian-H** [comment]
Das P.S. bezieht sich auf das eigene Problem, dass wir beim versenden Probleme hatten und deswegen dies ja auch in die Logfiles aufnahmen? War nur die Frage nach dem Zusammenhang. Da wir das Problem aber nicht mehr haben, ist "P.S.: .... " irrelevant.

**Comment by Christian-H** [comment]
![e-mail-sortierung-zeit-20150715](https://cloud.githubusercontent.com/assets/2972659/8699089/35338228-2b03-11e5-8b4f-11c36abebd50.jpg)
Hier ein Beispiel, wo die angezeigte Zeit seltsam ist. Die Header Zeit ist richtig. Von daher ist das hier nicht stringent. Hängt das mit dem Cron Job zusammen?

**Comment by dergel** [comment]
>Hängt das mit dem Cron Job zusammen? 

ja.. das sind die zeiten der email.. abgelegt werden die emails nach der zeit des herunterladens.
